src/frame/content.rs

```diff
@@ -36,6 +36,14 @@ impl Content {
         }
     }
 
+    /// Returns split values of the `Text` frame or None if the value is not `Text`.
```

Comment:
Could we have a little more information about how this differs from the regular `text` method?

A little excerpt from the spec about how values are separated will explain a lot! :)

Reply:
Added some more documentation